packages/libkcapi/libkcapi.spec

```diff
@@ -1,31 +1,29 @@
-# libkcapi since 85bce6035b (1.5.0) uses sha512hmac with the same key for all
-# self-checks. Earlier versions used sha256hmac with a different key to check
-# the shared library.
+# Helper functions that use OpenSSL to compute an HMAC with specified keys.
 %global openssl_sha512_hmac openssl sha512 -hmac FIPS-FTW-RHT2009 -hex
+%global openssl_sha256_hmac openssl sha256 -hmac orboDeJITITejsirpADONivirpUkvarP -hex
```

Comment:
Just for my own understanding, why this key?

Reply:
> Just for my own understanding, why this key?

Not sure of the origin; it's [the well-known key](https://github.com/search?q=orboDeJITITejsirpADONivirpUkvarP&type=code&p=1) for this.